src/main.rs

```diff
@@ -105,61 +105,60 @@ fn run(cli: Cli) -> Result<i32> {
     Ok(invocation.exit_code())
 }
 
-#[derive(Parser)]
-#[clap(setting = AppSettings::DeriveDisplayOrder)]
-#[clap(about, version)]
+#[derive(Debug, Parser)]
+#[command(about, version)]
 struct Cli {
     /// The command to run
-    #[clap(required = true, last = true)]
+    #[arg(required = true, last = true)]
     command: Vec<OsString>,
 
     /// Duration the cached result will be valid for
-    #[clap(long, default_value = "60s", visible_alias = "time-to-live", env = "BKT_TTL")]
+    #[arg(long, value_name = "DURATION", default_value = "60s", visible_alias = "time-to-live", env = "BKT_TTL")]
     ttl: humantime::Duration,
 
     /// Duration after which the result will be asynchronously refreshed
-    #[clap(long, conflicts_with = "warm")]
+    #[arg(long, value_name = "DURATION", conflicts_with = "warm")]
     stale: Option<humantime::Duration>,
 
     /// Asynchronously execute and cache the given command, even if it's already cached
-    #[clap(long)]
+    #[arg(long)]
     warm: bool,
 
     /// Execute and cache the given command, even if it's already cached
-    #[clap(long, conflicts_with = "warm")]
+    #[arg(long, conflicts_with = "warm")]
     force: bool,
 
     /// Includes the current working directory in the cache key,
     /// so that the same command run in different directories caches separately
-    #[clap(long, visible_alias = "use-working-dir")]
+    #[arg(long, visible_alias = "use-working-dir")]
     cwd: bool,
 
     /// Includes the given environment variable in the cache key,
     /// so that the same command run with different values for the given variables caches separately
-    #[clap(long, visible_alias = "use-environment")]
-    env: Option<Vec<OsString>>,
+    #[arg(long, value_name = "NAME", visible_alias = "use-environment")]
+    env: Vec<OsString>,
 
-    // Includes the last modification time of the given file(s) in the cache key,
+    /// Includes the last modification time of the given file(s) in the cache key,
     /// so that the same command run with different modtimes for the given files caches separately
-    #[clap(long, visible_alias = "use_file_modtime", multiple_occurrences=true, use_value_delimiter=false)]
```

Comment:
Can these settings be preserved?

Reply:
These are the default now

Reply:
Well gee, I guess [this line](https://github.com/dimo414/bkt/blob/4b6808d5d43543fa0a45356c5d58f24235abadd2/README.md?plain=1#L112) was never true.... :)